homeassistant/components/hausbus/gateway.py

```diff
@@ -0,0 +1,156 @@
+"""Representation of a Haus-Bus gateway."""
```

Comment:
this also looks like something that belongs in the library

Reply:
This is the part that actually translates to library events to home assistant specific stuff, so I'd say this should not be in the pyhausbus library.